arrays.js

```diff
@@ -145,14 +210,21 @@ Task 7:
 **************************************************************/
 function updateGrades(curve, students) {
   //TODO: Add your code here
+  students.map(function(o){
```

Comment:
It's not correct to mutate (change) the original array because 'map' creates a new array with changed values. We have to use the spread operator. Additionally, the solution misses a small step, which involves subtracting 'curve' from the 'nerds' type.